app/src/main/java/com/example/coopertest/SettingsActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.coopertest
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.edit
+//import androidx.preference.PreferenceFragmentCompat
+import com.example.coopertest.PrefsFragment as PreferenceFragmentCompat
+
+class SettingsActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.settings_activity)
+        supportFragmentManager
+            .beginTransaction()
+            .replace(R.id.settings, SettingsFragment())
```

Comment:
It shows a type error in the IDE for me, but the app still works :thinking:

Reply:
I think it's a bug because no error on mine.